prbt_support/urdf/prbt.xacro

```diff
@@ -59,8 +59,14 @@ limitations under the License.
     <!-- name of the gripper -->
     <xacro:property name="gripper_name" value="$(arg robot_prefix)gripper"/>
 
+    <!-- load gripper macro -->
     <xacro:include filename="$(find prbt_${gripper}_support)/urdf/${gripper}.urdf.xacro" />
 
+    <!-- instantiate gripper -->
+    <xacro:if value="${gripper == 'pg70'}">
+      <xacro:pg70 parent="$(arg robot_prefix)flange" />
```

Comment:
What about naming the macro `prbt_gripper` instead of hard-coding its name? Then approach would be generic: include `${gripper}.urdf.xacro` which defines the macro. Instantiate it with one generic macro name.